### PR TITLE
Applying Amazon Patch

### PR DESCRIPTION
**Pull request by anthonyhbo**
Applying Patch provided by Amzn.

### CODE REVIEW

File: library/src/main/java/com/google/android/exoplayer/MediaCodecAudioTrackRenderer.java [hunk 1]
```diff
@@ -314,7 +335,17 @@ private void seekToInternal(long positionUs) {
   protected boolean processOutputBuffer(long positionUs, long elapsedRealtimeUs, MediaCodec codec,
       ByteBuffer buffer, MediaCodec.BufferInfo bufferInfo, int bufferIndex, boolean shouldSkip)
       throws ExoPlaybackException {
-    if (shouldSkip) {
+    log.d("processOutputBuffer: positionUs = " + positionUs +
+            " elapsedRealtimeUs =  " + elapsedRealtimeUs +
+            " bufferInfo.flags = " + bufferInfo.flags +
+            " bufferIndex = " + bufferIndex +
+            " shouldSkip = " + shouldSkip +
+            " presentationTimeUs = " + bufferInfo.presentationTimeUs);
```
Comment (brandf) [inline library/src/main/java/com/google/android/exoplayer/MediaCodecAudioTrackRenderer.java]: Does this spam the log?  We may want to disable this if so.

**Comment by brandf** [comment]
This looks good. I did notice that they left a lot of logging in.  Not sure if that will be annoying, but we can disable as needed I suppose.

Is there a tool that processes // AMZN_CHANGE_ONELINE, AMZN_CHANGE_BEGIN, etc?

File: library/src/main/java/com/google/android/exoplayer/MediaCodecAudioTrackRenderer.java (hunk 1 above)
Comment (anthonyhbo) [inline library/src/main/java/com/google/android/exoplayer/MediaCodecAudioTrackRenderer.java]: Only if we use the logging set to debug.

**Comment by anthonyhbo** [comment]
We'll come back to this once we've gotten a patch which works with the older Fire TV

**Pull request merged**